cfgov/unprocessed/css/molecules/notification.less

```diff
@@ -90,6 +94,11 @@
     }
 }
 
+// If JavaScript is disabled, show the notification by default.
```

Comment:
If the server is including a notification at all but not including the `__visible` modifier, it means the notification will be shown when a runtime action triggers it. If, however, that's not possible due to JS being disabled (for whatever reason), the functionality for showing the notification will be disabled as well. In that case this is showing the notification so the server-side logic can still be guaranteed that it'll be displayed... however, thinking about that, I guess that doesn't make sense because the runtime action won't be available so the notification shown may not correspond to whatever the user is doing. You're right, so if JS isn't available, this just won't display. Fixed in https://github.com/cfpb/cfgov-refresh/commit/3aed7d500845b17fa48b1a9ece5cd8bb2d3459ea